ted2zim/scraper.py

```diff
@@ -508,11 +512,11 @@ def download_subtitles(self):
         # Download the subtitle files, generate a WebVTT file
         # and save the subtitles in
         # build_dir/{video id}/subs/subs_{language code}.vtt
-        self.load_meta_from_file()
-        for video in self.videos:
-            video_id = str(video["id"])
-            video_title = video["title"]
-            video_subtitles = video["subtitles"]
+        # self.load_meta_from_file()
+        for i in range(len(self.videos)):
```

Comment:
no reason to loop on that range here. `self.videos` is already in memory and data is not duplicated so just loop on `self.videos`.

Reply:
Used enumeration